### ISSUE TITLE
[Feature] Some features asked in Google Play reviews

### ISSUE THREAD

**Issue by TobiasBielefeld** [TobiasBielefeld/Simple-Solitaire#114]
* option to hide the in game menu
* option to deactivate the auto complete button for each game individually
* show the number of steps made in a  game
* make fullscreen possible on a galaxy s8+

**Comment by Hund** [comment]
I came here to request a option to about hiding the in-game menu. I see I'm not the only one.

It takes up valuable space on any medium to small screen. It would be nice to be able to hide it. :)

**Comment by TobiasBielefeld** [comment]
Yes the feature is nearly finished. Hiding the navigation bar will also be possible!
